changed bench32prefixes, gaiacli keys parse command still use cosmos … (#5092)
cosmos/cosmos-sdk: CHANGELOG.md

### Improvements

> * (client/keys) [\#5091](https://github.com/cosmos/cosmos-sdk/issues/5091) Changed prefixes using `SetBech32PrefixForAccount(),SetBech32PrefixForValidator(),SetBech32PrefixForConsensusNode()`method, but `gaiacli keys parse` command still use `cosmos`prefix.

Reviewer: ```suggestion
* (keys) [\#5091](https://github.com/cosmos/cosmos-sdk/issues/5091) Changed prefixes using `SetBech32PrefixForAccount(), SetBech32PrefixForValidator(), SetBech32PrefixForConsensusNode()` methods are respected in `keys parse`.
```
